example/src/main/tut/typeclass/Ord.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: docs
+title:  "Ord"
+---
+
+# Ord
+
+*Ord is used for totally ordered datatypes.*
```

Comment:
Maybe explain that "totally ordered" means that any two values must have a defined ordering between them? The type of `comp` does imply that, though.

Reply:
Is the explanation right below not sufficient?